src/Services/OData/ODataHandler.cs

```diff
@@ -59,17 +63,29 @@ static ODataHandler()
         internal const string PROPERTY_BINARY = "Binary";
         internal const int EXPANSIONLIMIT = Int32.MaxValue;
 
+        /// <inheritdoc select="summary" />
+        /// <remarks>Returns with false in this implementation.</remarks>
         public bool IsReusable
         {
             get { return false; }
         }
 
         internal ODataRequest ODataRequest { get; private set; }
 
+        /// <inheritdoc />
+        /// <summary>
+        /// Processes the OData web request.
+        /// </summary>
         public void ProcessRequest(HttpContext context)
         {
             ProcessRequest(context, context.Request.HttpMethod.ToUpper(), context.Request.InputStream);
         }
+        /// <summary>
+        /// Processes the OData web request. Designed for test purposes.
+        /// </summary>
+        /// <param name="context"></param>

```

Comment:
some description is missing for the context param

Reply:
ok